ezflow/functional/criterion/multiscale.py

```diff
@@ -57,6 +57,9 @@ def from_config(cls, cfg):
 
     def forward(self, pred, label):
 
+        if label.shape[1] == 3:
+            label = label[:, :2, :, :]
```

Comment:
Just so we remember why we did this, how about adding a comment here stating that the valid mask is being ignore?